zuul.d/templates.yaml

```diff
@@ -0,0 +1,12 @@
+- project-template:
+    name: python
+    check:
+      jobs:
+        - tox:
```

Comment:
This is a good start, but we should look to create new jobs in the long run, which can parent to tox and setup specific tox_envlist variables